style.css

```diff
@@ -0,0 +1,397 @@
+* {
+	box-sizing:border-box;
+	scroll-behavior: smooth;
+}
+
+.nav-link {
+	display: flex;
+	justify-content: flex-start;
+	gap: 20px;
+	align-items: center;
+}
+
+img[alt="french"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
+
+img[alt="UK"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
+.nav-link a {
+	text-decoration: none;
+	color:#FAFAFA;
+	font-weight: bold;
+}
+
+.nav-link a:hover {
+	color: #FF7E67;
+	display: flex;
+	height: 2rem;
+	align-items: center;
+	cursor: pointer;
+	font-size: 2rem;
+}
+
+.contact-button {
+	display: flex;
+	justify-content: flex-end;
+	gap: 20px;
+}
+
+.contact-button a {
+	text-decoration: none;
+	color:#FAFAFA;
+	font-weight: bold;
+	font-size: 2rem;
+}
+
+.contact-button a:hover{
+	color: #FF7E67;
+	display: flex;
+	height: 3rem;
+	align-items: center;
+	cursor: pointer;
+	font-size: 2rem;
+}
+
+nav {
+	align-items: center;
+	display:flex;
+	padding: 20px;
+	justify-content: space-between;
+	background-color:#3F5963;
+	width:100%;
+	top:0;
+	left:0;
+	right:0;
+	position: fixed;
+	border-bottom:3px solid #FF7E67;
+	font-family: "Work Sans", sans-serif;
+	text-transform: uppercase;
+	font-weight: bold;	
+}
+
+.intro_outerbox {
+	display:flex;
+	flex-direction: column;
+	padding: 40px;
+	max-width: 800px;
+}
+
+.intro_container {
+	display: flex;
+	margin: 20px 0 0;
+	font-family: "Work Sans", sans-serif;
+	font-weight: bold;
+	font-size: 3rem;
+	color: #FFFFFF;
+	background-color: #FF7E67;
+	padding: 5px;
+}
+
+.intro_container a {
+	text-decoration:none;
+	color: #FAFAFA;
+}
+
+.intro_container a:hover {
+	text-decoration:none;
+	color: #3F5963;
+}
+
+.intro {
+	font-family: "Work Sans", sans-serif;
+	font-size: 2rem;
+	border: 2px solid #FF7E67;
+	padding: 5px;
+	background-color: #FAFAFA;
+}
+
+header img {
+	width: 50%;
+	display: flex;
```

Comment:
display flex is for containers whereas this is just 1 image, so you wouldn't really put flex on it